### DISCUSSION TITLE
Code review strategy

### DISCUSSION THREAD

**Discussion by awvwgk** [fortran-lang/stdlib#447] [General]
We get quite a decent influx of new code contributions lately, which is really great and shows the general interest in stdlib. Still, I'm a bit worried that we can't keep up with the code review as the number of open pull requests grows.

Looking over the recently merged pull requests, there were about four to five maintainers active in reviewing, approving and merging PRs. For most submitted PRs finding reviewers seems to be the rate limiting step so far and we have PRs that have been stale for quite a while now due to lack of code reviews.

The obvious way forward on this would be to grow our reviewer pool. Maybe we can ask on the discourse for helping hands or try to reactivate maintainers that haven't been active in a while?

**Comment by Jim-215-Fisher** [comment]
Yes, we need recruit more reviewers. Some PRs have been there for more than 3 months, like stats_distribution_*. Can't continue further without clearing those out.

**Reply by awvwgk** [reply]
I would help with those but unfortunately this is hardly my field of expertise, sorry.

**Reply by epagone** [reply]
I feel partially responsible here: I have tested your uniform distribution module some time ago and it appears to be robust but I intended to stress-test it a bit more and never found the time to properly do it. Finally, I do not feel much qualified about the quality of implementation.

**Comment by awvwgk** [comment]
I added @gareth-nx' review guidelines to the wiki at https://github.com/fortran-lang/stdlib/wiki/Code-Review. The stdlib wiki should be world editable.

**Comment by awvwgk** [comment]
I haven't been actively watching the incoming patches at stdlib lately, as I want to shift my focus more to developing fpm for now.

Since it looks like we are again piling up on patches that need more eyes before getting merged, feel free to ping me for reviews in some of the open PRs and I'll try to spend some hours next weekend on reviews.
